docs/guides/internal.md

```diff
@@ -35,10 +35,10 @@ states: {
 },
 on: {
   // internal transitions, equivalent to the first example
-  LEFT_CLICK: { target: 'direction.left', internal: true },
-  RIGHT_CLICK: { target: 'direction.right', internal: true },
-  CENTER_CLICK: { target: 'direction.center', internal: true },
-  JUSTIFY_CLICK: { target: 'direction.justify', internal: true }
```

Comment:
These should be `'.left'`, `'.right'`, etc.

Reply:
I tested using the Visualizer, `{ target: 'center', internal: true }` has the same effect as `{ target: '.center', internal: true }`. Is that a bug?

Reply:
It might be. `target: 'center'` means "the sibling state with key of `"center"`."

Reply:
I think I got it. `.someState` is actually `currentStateNode.childState`. Am I wrong?

Reply:
You're correct. Let's open an issue (or better yet, a PR) for adding this to the docs here: https://xstate.js.org/docs/guides/ids.html